site/en/external/lockfile.md

```diff
@@ -224,3 +224,47 @@ practices:
 By following these best practices, you can effectively utilize the lockfile
 feature in Bazel, leading to more efficient, reliable, and collaborative
 software development workflows.
+
+## Merge Conflicts {:#merge-conflicts}
+
+The lockfile format is designed to minimize merge conflicts, but they can still
+happen.
+
+### Automatic Resolution {:#automatic-resolution}
+
+Bazel provides a custom
+[git merge driver](https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver)
+to help resolve these conflicts automatically.
+
+Set up the driver by adding this line to a `.gitattributes` file in the root of
+your git repository:
+
+```gitattributes
+MODULE.bazel.lock merge=bazel-lockfile-merge
```

Comment:
I'd suggest a comment in this snippet as well, so a user who copy-pastes the snippet to their repo will "accidentally" include some documentation for their co-workers to discover the install steps below.
In particular "each developer who wants to..." is a footgun and having a comment in the `.gitattributes` will help a little